src/App.tsx

```diff
@@ -8,37 +8,80 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { getUserPosts } from './api/posts';
+import { User } from './types/User';
+import { Post } from './types/Post';
 
 export const App: React.FC = () => {
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [loaded, setLoaded] = useState(false);
+  const [hasError, setError] = useState(false);
```

Comment:
Please pay attention to this rule
![image](https://user-images.githubusercontent.com/96409563/201747911-dce36a5b-fa94-4cbf-8f94-15b54318dec2.png)